deploy/ansible/roles/iroha-k8s/files/k8s-iroha.yaml

```diff
@@ -39,34 +59,80 @@ spec:
         env:
           - name: POSTGRES_PASSWORD
             value: mysecretpassword
+          - name: PGDATA
+            value: /var/lib/postgresql/data/pgdata
+        livenessProbe:
+          exec:
+            command:
+            - pg_isready
+            - -h
+            - localhost
+            - -U
+            - postgres
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+        readinessProbe:
+          exec:
+            command:
+            - pg_isready
+            - -h
+            - localhost
+            - -U
+            - postgres
+          initialDelaySeconds: 5
+          timeoutSeconds: 1
+        volumeMounts:
+        - name: pg-data
+          mountPath: /var/lib/postgresql/data
       - name: iroha
-        #image: hyperledger/iroha@sha256:eb059d2a03544e2bde91a0e16b7d03076b09bc31e9b970c1c4269f02084e8e91
-        image: hyperledger/iroha:develop
+        image: hyperledger/iroha:1.0.0_beta-4
```

Comment:
Hm, it might be a better idea to use image without any tag. Ping @bakhtin

Reply:
I've switched to a sha256 digest for the beta-4 tag. Additionally `iroha_image` can now be configured as the users discretion.